executing/_position_node_finder.py

```diff
@@ -291,8 +301,26 @@ def is_except_cleanup(inst: dis.Instruction, node: EnhancedAST) -> bool:
         #  except TypeError as msg:
         #      print("Sorry:", msg, file=file)
 
+        if (
+            isinstance(node, ast.Name)
+            and isinstance(node.ctx,ast.Store)
+            and inst.opname.startswith("STORE_")
+            and mangled_name(node) == inst.argval
+        ):
+            # This is a actually store to the variable
+            return False
+
+        if (
+            isinstance(node, ast.Name)
+            and isinstance(node.ctx,ast.Del)
+            and inst.opname.startswith("DELETE_")
+            and mangled_name(node) == inst.argval
+        ):
+            # This is a actually store to the variable
```

Comment:
1. Is this comment meant to be the same as the one above?
2. The comment doesn't seem gramatically correct in either case
3. These conditions could be combined into one, there's some duplication